### PR TITLE
fix(sonarr): Increase preferred and maximum Sonarr quality definition sizes

### PR DESCRIPTION
**Pull request by yammes08**
# Pull Request

## Purpose

To maintain compatibility with Sonarr after it merges an increase to the quality definition ranges - namely that `1000` is now 'Unlimited' where `400` was the previous value.

## Approach

In the Sonarr quality definitions JSON
- Change `395` to `995` for preferred quality
- Change `400` to `1000` for max quality

## Open Questions and Pre-Merge TODOs

None

## Requirements

- [x] These changes meet the standards for [contributing](https://github.com/TRaSH-Guides/Guides/blob/master/CONTRIBUTING.md).
- [x] I have read the [code of conduct](https://github.com/TRaSH-Guides/Guides/blob/master/.github/CODE_OF_CONDUCT.md).

### CODE REVIEW

**Comment by yammes08** [comment]
Can be merged once these changes land in Sonarr Master.

**Review by bakerboy448 — APPROVED** [review]

**Pull request merged**
